<launch>
  <node pkg="teleop_twist_monkey" type="monkey_node" name="monkey_node" output="screen">
    <param name="linear_min" value="-0.25"/>
    <param name="linear_max" value="0.25"/>
    <param name="angular_min" value="-0.5"/>
    <param name="angular_max" value="0.5"/>
    <param name="refresh_duration_x" value="5.0"/>
    <param name="refresh_duration_z" value="5.0"/>
    <param name="crossfade_coeff" value="0.01"/>
    <param name="publish_rate" value="20"/>
  </node>
</launch>
